src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/MessageBodyRetriever.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.blobrouter.servicebus;
+
+import com.microsoft.azure.servicebus.MessageBody;
+import org.springframework.util.CollectionUtils;
+
+import java.util.List;
+
+public final class MessageBodyRetriever {
```

Comment:
oh wait a sec, this is only used for testing right? blob router only sends messages. perhaps we can move this util class to test scope then ;)

Reply:
done